Python/PythonIntro.md

```diff
@@ -445,9 +445,43 @@ rover.__doc__
 dir(rover)
 # [__dir__, __doc__, ..., 'bark', 'barks', 'is_happy', 'name', 'owner', 'wagging']
 ```
+```
+Python String
+In Python, Strings are arrays of bytes representing Unicode characters. However, Python does not have a character data type, a single character is simply a string with a length of 1. Square brackets can be used to access elements of the string.
```

Comment:
Could you provide an example of using square brackets to access elements of a string? Also, maybe change "elements" to something else, something that makes sense that the elements you're accessing are either single characters or substrings.. "P